zebra-state/src/service.rs

```diff
@@ -1152,7 +1161,7 @@ impl Service<ReadRequest> for ReadStateService {
                         Ok(ReadResponse::Depth(depth))
                     })
                 })
-                .map(|join_result| join_result.expect("panic in ReadRequest::Tip"))
+                .map(|join_result| join_result.expect("panic in ReadRequest::Depth"))
```

Comment:
Nice catch!

We should refactor the repetitive the timer/spawn_blocking/span/map into a function later.